types/expo/index.d.ts

```diff
@@ -632,7 +633,32 @@ export interface BarCodeScannerProps extends ViewProperties {
     onBarCodeRead?: BarCodeReadCallback;
 }
 
-export class BarCodeScanner extends Component<BarCodeScannerProps> { }
+export class BarCodeScanner extends Component<BarCodeScannerProps> {
+  static Constants: {
```

Comment:
@fhelwanger hello, https://github.com/expo/expo-sdk/blob/master/src/CameraBasedBarCodeScanner.js#L19
BarCodeScanner Constants is extending CameraConstants. Can you extend or union this interface with CameraConstants? Like this -> https://github.com/DefinitelyTyped/DefinitelyTyped/pull/23812/commits/cec29159bd0034cbdc527d8e60b2eef1f153ade1#diff-61992a697c86241af33a41d9361c703c

Reply:
and interface props should separate by semicolon

Reply:
Hello! Sure, done.